### DISCUSSION TITLE
Bootstrap 5 adding badges to text within cell

### DISCUSSION THREAD

**Discussion by r-evans18** [rappasoft/laravel-livewire-tables#1260] [Q&A] [answered]
Hi there, anyway i can add badges inside of the table row cell around the value text which still allows it to be sortable and searchable?

If this can be done via a view please can you let me know how i can achieve this which still allows it to be searchable and sortable please.

Cheers

**Comment by lrljoe** [comment, marked as answer]
Yep, you can call format() on the column, and pass the value into the view:
```
Column::make('Name')
    ->format(
        fn($value, $row, Column $column) => view('my.custom.view')->withValue($value)
    ),
```

**Reply by r-evans18** [reply]
Thank you
